src/v2/features/vs/damage-calculator-service.tsx

```
@@ -0,0 +1,136 @@
+        if (type == undefined) return undefined;
+        return DamageType[type as keyof typeof DamageType];
+    }
+
+    private static getDamageFrom1Hit(damage: number, type: DamageType, armor: number, gravis: boolean): number {
+        const damageWithPierce = (damage * this.getPierce(type)) / 100.0;
+        const damageOverArmor = damage - armor;
+        const totalDamage = Math.round(Math.max(damageWithPierce, damageOverArmor));
+        if (gravis) return this.getDamageFrom1Hit(totalDamage, type, armor, false);
+        return totalDamage;
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)